protected/humhub/config/__autocomplete.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * This class only exists here for IDE (PHPStorm/Netbeans/...) autocompletion.
+ * This file is never included anywhere.
+ * Adjust this file to match classes configured in your application config, to enable IDE autocompletion for custom components.
+ * Example: A property phpdoc can be added in `__Application` class as `@property \vendor\package\Rollbar|__Rollbar $rollbar` and adding a class in this file
+ * ```php
+ * // @property of \vendor\package\Rollbar goes here
+ * class __Rollbar {
+ * }
+ * ```
+ */
+class Yii {
```

Comment:
We already have something like this here:
https://github.com/humhub/humhub/blob/master/protected/humhub/Yii.php

I also like the approach of having this in the config folder.

Reply:
Okay. I'll move it to config folder and will change it according new autocomplete standart like this https://github.com/yiisoft/yii2-app-basic/blob/master/config/__autocomplete.php